auditlog/registry.py

```diff
@@ -148,4 +154,75 @@ def _dispatch_uid(self, signal, model) -> DispatchUID:
         return self.__hash__(), model.__qualname__, signal.__hash__()
 
 
+def _get_model_classes(app_model: str) -> List[ModelBase]:
+    try:
+        try:
+            app_label, model_name = app_model.split(".")
+            return [apps.get_model(app_label, model_name)]
+        except ValueError:
+            return apps.get_app_config(app_model).get_models()
+    except LookupError:
+        return []
+
+
+def _auditlog_register_models(
+    auditlog: AuditlogModelRegistry, models: Iterable[Union[str, Dict[str, Any]]]
+):
+    models = copy.deepcopy(models)
+
+    if not isinstance(models, list) and not isinstance(models, tuple):
+        raise TypeError(f"'models'({type(models)}) is not an list or tuple")
+
+    for model in models:
+        if isinstance(model, str):
+            for model_class in _get_model_classes(model):
+                auditlog.unregister(model_class)
+                auditlog.register(model_class)
+
+        elif isinstance(model, dict):
+            if "model" not in model:
+                raise ValueError("item must contain 'model' key")
+            if "." not in model["model"]:
+                raise ValueError(
+                    "model with options must be in the format <app_name>.<model_name>"
+                )
+            try:
+                model["model"] = _get_model_classes(model["model"])[0]
+                auditlog.unregister(model["model"])
+                auditlog.register(**model)
+            except IndexError:
+                pass
```

Comment:
This line here doesn't have test coverage

Reply:
@hramezani 
I think IndexError is never raised. So I deleted that statement.